website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+Will human organs [scope of organs required; enhanced organ function or treatment for organ inadeqaucy?] grown from genetically modified cells be widely used [scope of widely used required] for transplants by 20XX?
+
+Will more than XX% of all cancer therapies [scope of cancer therapies required] involve some form of DNA or genetic engineering by 20XX?
+
+Will gene-editing [scope of gene-editing required] be used to treat autoimmune diseases [diseases required, e.g., Type 1 Diabetes, Rheumatoid Arthritis] in clinical practice [scope of clinical practice required] by 20XX?
+
+Will artificial intelligence be used to automatically [definition of automatically required] design personalized gene therapies [scope of gene therapies required] for patients [patient and or newborns?] by 20XX?
+
+Will a clinical trial [scope of trial required] using gene-editing to treat or prevent aging-related diseases be conducted by 20XX?
+
+Will there be a clinical trial [scope of trial required] using gene-editing for the enhancement of intelligence by 20XX?
+
+Will gene therapies to prevent inherited genetic disorders [disorder scope required, e.g. cystic fibrosis, sickle cell anemia, ...] be standard [definition of standard required] in newborn care in OECD countries [country designation required] by 20XX?
+
+Will more than XX% of hospitals [possibly poor measure for question] in OECD countries [country designation required] use gene-editing to treat infectious diseases [disease scope required, e.g., HIV, hepatitis] by 20XX?
+
+Will polygenic scoring be used to identify genetic predisposition to addiction [check required to see if this has already occurred, if so convert to continuous question] in clinical settings by 20XX?
+
+Will gene-editing be used in clinical trials [scope of trial required] to enhance [scope of enhancement required] immune responses in humans by 20XX?
+
+Will gene-editing be used to increase resistance to viral infections [infection scope required, e.g., influenza, COVID] by 20XX?
+
+Will gene-editing treatments for genetic vision impairments [impairment scope required, e.g., retinitis pigmentosa] be clinically approved [scope of approval required] by 20XX?
+
+Will polygenic risk scoring for mental health [scope of mental health required] be used in over XX% of hospitals [again, hospitals possibly not the best measure] by 20XX?
+
+Will polygenic embryo screening become a standard method [detailed definition of standard required] for selecting embryos in fertility clinics by 20XX?
+
+[question in general needs to be more specific, but hopefully the idea is communicated] Will synthetic DNA or gene circuits be used in human tissues for drug delivery or therapeutic purposes by 20XX?
+
+In what year will gene-editing therapies [scope of therapy and age-of-application required] for muscular dystrophy be available in over XX% of countries?
+
```

Comment:
why do these questions about specific disorders all involve slightly different criteria? shouldn't they be standardized to provide a cross-section of the progress of gene engineering techniques in disorders of particular types?